codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/message/XMLMessageMarshallableGenerator.kt

```diff
@@ -1,182 +0,0 @@
-package software.amazon.smithy.aws.swift.codegen.message
-
-import software.amazon.smithy.codegen.core.CodegenException
-import software.amazon.smithy.codegen.core.Symbol
-import software.amazon.smithy.model.shapes.MemberShape
-import software.amazon.smithy.model.shapes.Shape
-import software.amazon.smithy.model.shapes.ShapeType
-import software.amazon.smithy.model.shapes.UnionShape
-import software.amazon.smithy.model.traits.EventHeaderTrait
-import software.amazon.smithy.model.traits.EventPayloadTrait
-import software.amazon.smithy.swift.codegen.ClientRuntimeTypes
-import software.amazon.smithy.swift.codegen.SwiftDependency
-import software.amazon.smithy.swift.codegen.SwiftWriter
-import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-import software.amazon.smithy.swift.codegen.integration.serde.readwrite.DocumentWritingClosureUtils
-import software.amazon.smithy.swift.codegen.integration.serde.readwrite.WritingClosureUtils
-import software.amazon.smithy.swift.codegen.model.eventStreamEvents
-import software.amazon.smithy.swift.codegen.model.hasTrait
-
-class XMLMessageMarshallableGenerator(
```

Comment:
See `MessageMarshallableGenerator` above.  Don't know why Git didn't track the rename.